stream.js

```diff
@@ -37,6 +37,10 @@ page.onError = function (err, trace) {
 	console.error('WARN: ' + err + formatTrace(trace[0]));
 };
 
+page.onResourceError = function (resourceError) {
+	console.error('WARN: Unable to load resource #' + resourceError.id + ' → ' + resourceError.url);
```

Comment:
Cool – I've pushed an updated version that includes `errorString`.

Let me know if there's anything else I can do.